app/src/main/java/com/example/babytracking/MainActivity.kt

```diff
@@ -13,6 +14,7 @@ class MainActivity : AppCompatActivity() {
 
 
         val baby = findViewById(R.id.text_baby) as TextView
```

Comment:
Consider using View Bindings to assign your views, it's clearer: https://developer.android.com/topic/libraries/view-binding